reference/reflection/versions.xml

```diff
@@ -0,0 +1,446 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- $Revision$ -->
+<!--
+  Do NOT translate this file
+-->
+<versions>
```

Comment:
DO NOT TRANSLATE THIS FILE.

The doc tooling will automatically pull this from doc-en.